test/unit/spec/nbextensions/bulkImportCell/bulkImportCell-spec.js

```diff
@@ -304,6 +261,178 @@ define([
                     })
                 ).toThrow();
             });
+
+            describe('xsv specific setup', () => {
+                const inFiles = ['dvh.fasta', 'styphi.fasta', 'mex.fasta'],
```

Comment:
Mostly the same as the moved test, just added multiple cases to check that file order is maintained, and the warning message shows up as expected.